api/tests/unittest/middleware/test_request_context.py

```diff
@@ -54,45 +54,3 @@ def test_get_request_context(self):
         request_context = RequestContext(MagicMock())
         _request_context.set(request_context)
         self.assertEqual(request_context, get_request_context())
-
-    def test_is_user_email_restricted(self):
```

Comment:
This test just needs more time to be fixed, so I'm creating an issue for it and de-scoping it from this PR. Follow up issue, #849